medusa/providers/torrent/json/btn.py

```diff
@@ -171,36 +177,28 @@ def _check_auth_from_data(parsed_json):
     @staticmethod
     def _process_title_and_url(parsed_json):
 
-        # The BTN API gives a lot of information in response,
-        # however this application is built mostly around Scene or
-        # release names, which is why we are using them here.
+        # Try to get the release name,
+        # if it doesn't exist make one up
+        # from the proprieties obtained.
 
-        if 'ReleaseName' in parsed_json and parsed_json['ReleaseName']:
-            title = parsed_json['ReleaseName']
-
-        else:
+        title = parsed_json.get('ReleaseName')
+        if not title:
             # If we don't have a release name we need to get creative
             title = ''
             if 'Series' in parsed_json:
-                title += parsed_json['Series']
+                title += parsed_json.get('Series')
```

Comment:
since you're already checking `if 'Series' in parsed_json:`
then you can just use:
`title += parsed_json['Series']`

Same applies to the fields bellow